readthedocs/proxito/tests/test_full.py

```diff
@@ -366,7 +367,24 @@ def test_project_nginx_serving_unicode_filename(self):
         )
 
     @override_settings(PYTHON_MEDIA=False)
-    def test_download_files(self):
+    def test_download_files_public_version(self):
+        for type_ in DOWNLOADABLE_MEDIA_TYPES:
+            resp = self.client.get(
+                f"/_/downloads/en/latest/{type_}/",
+                HTTP_HOST="project.dev.readthedocs.io",
+            )
+            self.assertEqual(resp.status_code, 200)
+            extension = "zip" if type_ == MEDIA_TYPE_HTMLZIP else type_
```

Comment:
This seems like a code smell... Probably should have named `htmlzip` as `zip`... 🙃